pytorch_forecasting/models/base/_base_model_v2.py

```diff
@@ -294,3 +294,175 @@ def log_metrics(
                 prog_bar=True,
                 logger=True,
             )
+
+    def standardize_model_output(
+        self,
+        prediction: torch.Tensor,
+        expected_dims: tuple[int, Optional[int], Optional[int], Optional[int]] = (
+            None,
+            None,
+            None,
+            None,
+        ),  # noqa: E501
+    ) -> torch.Tensor:
+        """
+        Standardize model outputs to a 4-dimensional tensor, with shape
+        (batch_size, timesteps, num_features, last_dim).
+
+        Parameters
+        ----------
+        prediction : torch.Tensor
+            The raw prediction tensor from the model.
+            - Must be a torch.Tensor (in the future, also accept a list of tensors for
+              multi-target forecasting).
+            - Supported dims: 2D, 3D or 4D tensors.
+            - if 2D: (batch_size, timesteps) - univariate forecasting
+            - if 3D:
+                a) (batch_size, timesteps, n_targets) - multivariate forecasting
+                b) (batch_size, timesteps, last_dim) - univariate forecasting with quantiles or distribution.
+                c) (batch_size, timesteps, n_targets * last_dim) - multivariate
+              forecasting with quantiles, where features and quantiles are flattened in dim 2.
+            - if 4D: (batch_size, timesteps, n_targets, last_dim) - multivariate
+              forecasting with quantiles or distribution parameters.
+            - In the future, once multi-target forecasting is supported, this
```

Comment:
wait, not sure if I understand this. Is multi-target forcasting not just the case `n_targets>1`?